### ISSUE TITLE
Deploy the factory to all Etherscan supported chains 

### ISSUE THREAD

**Issue by 0xV4L3NT1N3** [ZeframLou/create3-factory#2]
Referencing supported networks from [Hardhat](https://github.com/NomicFoundation/hardhat/blob/main/packages/hardhat-etherscan/src/ChainConfig.ts).

**Comment by tab00** [comment]
I agree. @ZeframLou you should deploy the factory to many other networks, including testnets e.g. Polygon, Avalanche.

**Comment by wei3erHase** [comment]
+ Optimism Goerli 🙏

**Comment by CodeSandwich** [comment]
You may be interested in this comment: https://github.com/ZeframLou/create3-factory/issues/4#issuecomment-1640992276
